docs/migration/guide/how-to-set-up-docs-previews.md

```diff
@@ -25,21 +25,25 @@ This way you only build and deploy the docs when there are changes to the docs a
 name: docs-build <1>
 
 on:
+  push: <2>
+    branches: 
+        - main 
```

Comment:
@bmorelli25 this is the change needed in elastic/docs-content

Reply:
Nice! Thank you